### ISSUE TITLE
Builds are failing because of missing package.json

### ISSUE THREAD

**Issue by SamuelDebruyn** [bitrise-steplib/steps-npm#8]
Since v1.0.0, my builds have started failing with the following message:

```
Failed to download step.zip: Failed to download target from: https://bitrise-steplib-collection.s3.amazonaws.com/step-archives/npm/1.0.0/step.zip
+------------------------------------------------------------------------------+

| (4) Install appcenter-cli                                                    |
+------------------------------------------------------------------------------+
| id: npm                                                                      |
| version: 1.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2018-11-21T07:54:31-08:00                                              |
+------------------------------------------------------------------------------+
|                                                                              |
Config:
- Workdir: /Users/vagrant/git
- Command: install -g appcenter-cli
- NpmVersion: 
Autodetecting npm version
Checking package.json for npm version
error reading npm version from package.json: package.json file read error: No file found at path: /Users/vagrant/git/package.json
|                                                                              |
+---+---------------------------------------------------------------+----------+
| x | Install appcenter-cli (exit code: 1)                          | 14 sec   |
+---+---------------------------------------------------------------+----------+
| Issue tracker: https://github.com/bitrise-steplib/steps-npm/issues           |
| Source: https://github.com/bitrise-steplib/steps-npm                         |
+---+---------------------------------------------------------------+----------+
```

**Comment by bitce** [comment]
Hi @SamuelDebruyn sorry to hear about this! Can you please include a build URL?

**Comment by bitce** [comment]
Hi again! Just an update, we looked into the step and found the issue. Sorry for the inconvenience, should be fixed shortly, I will update this thread once it's done!

**Comment by Liliano1044** [comment]
Hi! Same error for me for these builds : 

https://app.bitrise.io/build/b001f8013a8bf96b
https://app.bitrise.io/build/b1e0322e14dea492
https://app.bitrise.io/build/cd316e0ea5ec3ab0

Bitrise Log :
INFO[09:46:29] Start installing (nodejs) with apt-get       
+------------------------------------------------------------------------------+

| (2) Installation de newman                                                   |
+------------------------------------------------------------------------------+
| id: npm                                                                      |
| version: 1.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2018-11-22T09:46:29Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
INFO[09:46:29]  * [OK] Step dependency (nodejs) installed, available. 
Config:
- Workdir: /bitrise/src
- Command: install -g newman
- NpmVersion: 
Autodetecting npm version
Checking package.json for npm version
error reading npm version from package.json: package.json file read error: No file found at path: /bitrise/src/package.json
|                                                                              |
+---+---------------------------------------------------------------+----------+
| x | Installation de newman (exit code: 1)                         | 2.31 sec |
+---+---------------------------------------------------------------+----------+
| Issue tracker: https://github.com/bitrise-steplib/steps-npm/issues           |
| Source: https://github.com/bitrise-steplib/steps-npm                         |
+---+---------------------------------------------------------------+----------+

**Comment by bitce** [comment]
Thanks for your report as well @Liliano1044 !

**Comment by Liliano1044** [comment]
Hi! It's my solution too for the moment : use  0.9.1 version instead of the latest by default

**Comment by bitce** [comment]
Hi @SamuelDebruyn and @Liliano1044 !

Thanks for your reports and patience! The issue should be fixed, a new step version was deployed, please let us know if you're still experiencing it!

**Comment by Liliano1044** [comment]
Hi again! Fix is OK. Thx.

just this thing :
 in log, there is this line just before the npm step:
"Failed to download step.zip: Failed to download target from: https://bitrise-steplib-collection.s3.amazonaws.com/step-archives/npm/1.0.1/step.zip"

Here is he build for example : https://app.bitrise.io/build/7109baacb45dcaa5

**Comment by lszucs** [comment]
Hey @SamuelDebruyn,

We have just published version 1.0.1, which fixes issues when there is no package.json in the working directory.

For more info, see: https://github.com/bitrise-steplib/steps-npm/releases/tag/1.0.1

For now, I'll close this issue, but if you need more info/help feel free to open a new issue!
